### PR TITLE
FUM-3164 Pass cluster info to app deployment

### PR DESCRIPTION
**Pull request by Davidoutz**

### CODE REVIEW

File: deploy/application/main.tf [hunk 1]
```diff
@@ -31,3 +31,12 @@ resource "helm_release" "this" {
     })
   ]
 }
+
+data "aws_ssm_parameters_by_path" "platform" {
+  path = "/platform/"
```
Comment (rbapst-tamedia) [inline deploy/application/main.tf:36]:
```suggestion
  path = "/platform/${var.eks_name}*" 
```
with eks_name a variable with default to ""

Comment (rbapst-tamedia) [reply deploy/application/main.tf:36]: (to test, I don't know if wildcards work or if path="/platform/xyz" will return all "/platform/xyz-123", "/platform/xyz-456", ...)

**Review by rbapst-tamedia — COMMENTED** [review]
I've made some test and it probably won't work because when you add a parameter in the parameter stores, the data....names are ordered alphabetically, so if you have the following parameters:
- /platform/rba-123/name
- /platform/rba-456/name
and you add a new value for rba-123, example:
- /platform/rba-123/input
Then the "/platorm/rba-123/input" will be the first in the "names" index :-(

File: deploy/application/main.tf (hunk 1 above)
Comment (rbapst-tamedia) [reply deploy/application/main.tf:36]: It's not working :-(

**Review by swibrow — COMMENTED** [review]
This probably works but I think we need to simplify it. If its not possible, then we can add some additional info to he param. eg timestamp

**Review by swibrow — APPROVED** [review]
it if works, lets go with it for now.

**Pull request merged**